src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -153,6 +153,11 @@ public void setPerson(Person target, Person editedPerson) {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void sortPersonListByDate() {
+            throw new AssertionError("This method should not be called.");
+        }
```

Comment:
May I know why is this method overridden but should never be called? Unsure why it's under AddCommandTest... 
Would there be a SortCommandTest.java in the future commit?

Reply:
Okay, this is because AddCommandTest has ModelStub that implements Model so since I created the sortPersonListByDate() method in the Model interface, ModelStub needs to have that method. That's why if you look at the other methods in ModelStub it also has the same AssertionError since

And yes I will implement a SortCommandTest in the future but I was focusing on pushing this out and ensuring HustleBook is stable before the demo during tutorial

Reply:
Okay that makes sense 😄 

LGTM! 💯